client/src/components/ImageGallery/Swiper.scss

```diff
@@ -67,7 +67,7 @@
 .c {
     position: relative;
     max-width: 1200px;
-    margin: 0 auto;
+    margin: 100px auto  0 auto;
```

Comment:
why are you constantly changing values and styling? that is causing lots of confusion in your branch, we could keep doing this forever )